### DISCUSSION TITLE
IPVM In-Person Workshop Q2'23

### DISCUSSION THREAD

**Discussion by expede** [ipvm-wg/spec#17] [Workshop]
For those that are in town for IPFS Thing in Brussels, we're going to get together in a room with a whiteboard!

RSVP here: https://lu.ma/ipvm-workshop-q2-2023

**Comment by mikevoronov** [comment]
The link seems broken, when is this workshop going to be organized?

**Reply by expede** [reply]
It was going to be here at IPFS Thing, but there wasn't enough overlap in attendees, so we're going to reschedule. In the meantime, we're going to run an unconf session later this week at IPFS Thing, though :)

**Reply by mikevoronov** [reply]
Got it, thanks, I'd like to attend this session :)

**Reply by expede** [reply]
@mikevoronov it's scheduled for Tuesday :)
![image](https://user-images.githubusercontent.com/1052016/232320987-8bc2872b-d6c4-4908-974a-b9d2e2088d31.png)

**Reply by mikevoronov** [reply]
I will attend, thank you!
